league/flysystem-bundle/1.0/config/packages/flysystem.yaml

```diff
@@ -0,0 +1,6 @@
+flysystem:
+    storages:
+        default.storage:
+            adapter: 'local'
+            options:
+                directory: '%kernel.project_dir%/storage'
```

Comment:
If this is `'%kernel.project_dir%/var/storage'` there is a good change that folder is writable already.

Reply:
Isn't there a habit of considering the var directory temporary and removable? I like the idea, but I'm not sure what people expect from var.